capstone/src/main/java/com/google/step/data/RestaurantScore.java

```diff
@@ -0,0 +1,46 @@
+package com.google.step.data;
+
+/**
+ * Class that holds the score for a specific restaurant.
+ */
+public class RestaurantScore {
+
+    // The name should always remain the same but the score will change when calculating it.
+    private final long restaurantKey;
+    private double score;
```

Comment:
Hmmm, is there a reason to construct this class before you have your final score? If you're computing a bunch of stuff all together, it might make sense to just have it in a map or keys to scores, and then if you're returning this individual class, you can return an immutable instead.

I'm not sure what your score calculation looks like yet, so I'm not sure what's the best final representation for this stuff at this time.

Megan's comment about just using the map also makes a lot of sense.

Reply:
Alright so I changed everything to be final and will be changing the client accordingly.